packages/itwinui-react/src/core/InputWithDecorations/InputWithDecorations.tsx

```diff
@@ -81,7 +81,7 @@ const InputWithDecorationsButton = React.forwardRef((props, ref) => {
  *
  * If you are not using default InputWithDecorations.Icon and InputWithDecorations.Button, use borderless versions of other components.
  *
- * @usage
+ * @example
  * <InputWithDecorations>
  *    <InputWithDecorations.Input />
  *    <InputWithDecorations.Icon>
```

Comment:
Can we also fix `InputWithDecorations.Icon` (nonexistent) to be `Icon`?

Reply:
Fixed (https://github.com/iTwin/iTwinUI/pull/1876/commits/314fe8af5de7f823978dd1aa1e9146b1d4afc266)

Reply:
The text in the paragraph above also includes a reference to it

Reply:
Removed that reference too (https://github.com/iTwin/iTwinUI/pull/1876/commits/2ae8f1ffc899f71fc3a03ab37756b4554a1ffdf9). Did a global search to confirm that there are no more occurrences.